controls/radrichtextbox/document-elements/table/working-with-tables.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Working with Tables in Code
+page_title: Working with Tables in Code
+description: Working with Tables in Code
+slug: radrichtextbox-working-with-tables
+tags: table
+published: True
+position: 1
+---
+
+# Working with Tables in Code
+
+This article shows you can create or retrieve tables in the code behind. 
```

Comment:
This article shows how you can create or retrieve tables in the code behind. 
Also, might be helpful to add here a link to the topic for the properties and methods related to tables